src/test/clojure/clara/test_keyword_names.cljc

```diff
@@ -0,0 +1,58 @@
+(ns clara.test-keyword-names
```

Comment:
It is possible that I'm missing something, but I don't think this file will actually be used in ClojureScript since it is in the src/test/clojure directory and it [looks from the project.clj](https://github.com/cerner/clara-rules/blob/0.17.0/project.clj#L45) like only src/test/clojurescript and src/test/common will be used in ClojureScript tests.

Reply:
I think that was left over from when I was trying to get the test to run in both CLJ and CLJS. I'll remove it.